<?xml version="1.0"?>
<launch>
    <node name="head_camera" pkg="usb_cam" type="usb_cam_node" output="screen" >
        <param name="video_device" value="/dev/video1" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="autofocus" type="bool" value="true" />
        <param name="camera_frame_id" value="head_camera_tf" />
        <param name="io_method" value="mmap"/>
    </node>
    <node name="alvar_logger" pkg="alvar_logger" type="alvar_logger" output="screen">
        <remap from="~input_image" to="/head_camera/image_raw"/>
        <param name="camera_info_topic" value="/head_camera/camera_info" />
        <param name="log_file" value="$(arg log_file)" />
        <param name="video_file" value="$(arg video_file)" />
    </node>
</launch>
